nodeup/pkg/bootstrap/install.go

```diff
@@ -44,35 +44,16 @@ func (i *Installation) Run() error {
 		return fmt.Errorf("error determining OS distribution: %v", err)
 	}
 
-	tasks := make(map[string]fi.NodeupTask)
+	tasks := make(map[string]fi.InstallTask)
 
-	buildContext := &fi.NodeupModelBuilderContext{
+	buildContext := &fi.InstallModelBuilderContext{
 		Tasks: tasks,
 	}
 	i.Build(buildContext)
 
-	// If there is a package task, we need an update packages task
```

Comment:
I guess the idea is that we know there isn't a Package task?  That makes sense if so!

Reply:
There's only a File and a Service.